.github/workflows/preview-comment.yml

```diff
@@ -0,0 +1,31 @@
+name: Comment on PR
+
+on:
+  pull_request:
+    branches:
+      - trunk
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ env.BOT_TOKEN }}
```

Comment:
We shouldn't need to supply a token. The default `$GITHUB_TOKEN` should be sufficient.

Reply:
Not with the `pull_request` trigger. The token is read-only. With `pull_request_target` it should work.

Alternatively a custom fine-grained PAT would need to be provided.

Reply:
I updated it to use the `pull_request_target`, thanks!